src/cleanlab_tlm/utils/vpc/chat_completions.py

```diff
@@ -0,0 +1,58 @@
+import os
+from typing import TYPE_CHECKING, Any, Optional
```

Comment:
dont forget to add docstring up here that clarifies this module is just for VPC users of TLM